shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -51,10 +52,16 @@ void EmbedderExternalViewEmbedder::CancelFrame() {
 
 // |ExternalViewEmbedder|
 void EmbedderExternalViewEmbedder::BeginFrame(
-    SkISize frame_size,
     GrDirectContext* context,
-    double device_pixel_ratio,
-    fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger) {
+    fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger) {}
+
+// |ExternalViewEmbedder|
+void EmbedderExternalViewEmbedder::PrepareView(int64_t native_view_id,
+                                               SkISize frame_size,
+                                               double device_pixel_ratio) {
+  // TODO(dkwingsmt): This class only supports rendering into the implicit view.
+  // Properly support multi-view in the future.
```

Comment:
Do we have a tracking issue for this? I'm assuming this affects macOS, so we'll want to fix this as part of the desktop multi-view project.

(In theory we should have a tracking issue for `IOSExternalViewEmbedder::PrepareView` too but I don't feel as strongly about that one!)

Reply:
Yeah I'll link https://github.com/flutter/flutter/issues/135530 item 4 to it. (Although I'll submit the PR for embedder API that removes this TODO.)